### PR TITLE
Trello-850 Update curator to work with ES 5.x

### PR DESCRIPTION
**Pull request by bmcelvee**
https://trello.com/c/xPBjqrcJ/850-document-update-curator-to-work-with-es-5x-epic-ois-es5logging

### CODE REVIEW

**Comment by bmcelvee** [comment]
@josefkarasek PTAL

I left out the 'Control cronjob' section from the repo, should that be added as well?

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -209,7 +209,7 @@ secret to be used for pulling component images from an authenticated registry.
 | The minute of the hour Curator will run.
 
 |`openshift_logging_curator_run_timezone`
-|The timezone Curator uses for figuring out its run time. Provide a the 
+|The timezone Curator uses for figuring out its run time. Provide a the
```
Comment (juzhao) [inline install_config/aggregate_logging.adoc]: Provide a the -> Provide a

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -1273,16 +1273,6 @@ logs, use the name `.operations` as the project name.
 |Use `.defaults` as the `$PROJECT_NAME` to set the defaults for projects that are
 not specified.
 
-|`runhour`
```
Comment (juzhao) [inline install_config/aggregate_logging.adoc]: @josefkarasek 
Should we delete runhour, runminute, and timezone variables from doc?

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -1319,19 +1311,20 @@ config.yaml: |
...
+        days: 2
 ----
 
+[NOTE]
+====
+The timezone is set based on the {product-title} master node.
+====
 
 [IMPORTANT]
 ====
```
Comment (juzhao) [inline install_config/aggregate_logging.adoc]: When you use `month` as the `$UNIT` for an operation
change to
When you use `months` as the `$UNIT` for an operation

File: install_config/aggregate_logging.adoc (hunk 2 above)
Comment (josefkarasek) [reply install_config/aggregate_logging.adoc]: @juzhao you're right. We should keep `runhour` and `runminute` as they allow users to specify at what part of the day the job will be scheduled.

The `timezone` timezone variable is no longer configurable as this value is sourced from Openshift master nodes.

File: install_config/aggregate_logging.adoc [hunk 4]
```diff
@@ -1354,28 +1416,42 @@ Curator. Currently the `logging-curator` ConfigMap is used to configure both
...
+. For scripted deployments, copy the configuration file that was created by the
+installer and create your new OpenShift custom config:
 +
 ----
-$ create /path/to/mycuratorconfig.yaml
-$ oc create configmap logging-curator -o yaml \
-  --from-file=config.yaml=/path/to/mycuratorconfig.yaml | \
-  oc replace -f -
+$ oc extract configmap/logging-curator --keys=curator5.yaml,config.yaml --to=/my/config
+$ edit /my/config/curator5.yaml
```
Comment (juzhao) [inline install_config/aggregate_logging.adoc]: edit is not one command
shall we delete the "$"?

File: install_config/aggregate_logging.adoc [hunk 5]
```diff
@@ -1345,6 +1338,75 @@ from that date. If you want to be exact with Curator, it is best to use days
...
+==== Using the Curator Actions File
+
+Setting the {product-title} custom configuration file format ensures internal
+indices are not mistakenly deleted.
+
+To use the *actions file*, add an exclude rule to your configuration to retain
+these indices. You must manually add all of the required patterns.
+
+----
+config.yaml: |
```
Comment (juzhao) [inline install_config/aggregate_logging.adoc]: @josefkarasek 
the action file should be put under actions.yaml, not config.yaml

File: install_config/aggregate_logging.adoc (hunk 3 above)
Comment (bmcelvee) [reply install_config/aggregate_logging.adoc]: Thanks, changed!

File: install_config/aggregate_logging.adoc (hunk 4 above)
Comment (bmcelvee) [reply install_config/aggregate_logging.adoc]: I updated this. Is it correct?

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL :european_castle:

File: install_config/aggregate_logging.adoc [hunk 6]
```diff
@@ -1315,6 +1312,8 @@ For example, to configure Curator to:
 - delete *operations* logs older than `8 weeks`
 - delete all other projects indices after they are `31 days` old
 - run the Curator jobs at midnight every day
+- delete indices older than 1 day that are matched by the '^project\..+\-dev.*$' regex
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: IBM Style implies that the first letter in each li should be capitalized.

File: install_config/aggregate_logging.adoc [hunk 7]
```diff
@@ -1370,28 +1439,42 @@ Curator. Currently the `logging-curator` ConfigMap is used to configure both
...
+. To create the Curator configuration, edit the configuration in the deployed
+ConfigMap:
 +
 ----
 $ oc edit configmap/logging-curator
 ----
 
-. To replace the provided ConfigMap instead:
+. For scripted deployments, copy the configuration file that was created by the
+installer and create your new OpenShift custom config:
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: new {product-title} custom configuration:

File: install_config/aggregate_logging.adoc [hunk 8]
```diff
@@ -1335,23 +1334,24 @@ config.yaml: |
...
       delete:
-        days: 15
+        days: 1
     - pattern: '^project\..+\-test\..*$'
       delete:
-        days: 30
+        days: 2
 ----
 
+[NOTE]
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: I might move this note to line 1252 and s/timezone/Curator time zone (The CCS glossary says "time zone" is preferred.)

File: install_config/aggregate_logging.adoc [hunk 9]
```diff
@@ -1370,28 +1439,42 @@ Curator. Currently the `logging-curator` ConfigMap is used to configure both
...
+  edit /my/config/config.yaml
+$ oc delete configmap logging-curator ; sleep 1
+$ oc create configmap logging-curator \
+    --from-file=curator5.yaml=/my/config/curator5.yaml \
+    --from-file=config.yaml=/my/config/config.yaml \
+    ; sleep 1
 ----
 
-. After you make your changes, redeploy Curator:
+. Or if you are using the *actions file*:
```
Comment (ahardin-rh) [inline install_config/aggregate_logging.adoc]: s/Or if you are using/Alternatively, if you are using

File: install_config/aggregate_logging.adoc [hunk 10]
```diff
@@ -1361,6 +1361,75 @@ from that date. If you want to be exact with Curator, it is best to use days
 (for example, `delete: days: 30`).
 ====
 
+[[aggregate-logging-using-curator-actions-file]]
+==== Using the Curator Actions File
+
+Setting the {product-title} custom configuration file format ensures internal
+indices are not mistakenly deleted.
+
+To use the *actions file*, add an exclude rule to your configuration to retain
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: maybe s/configuration/Curator configuration

File: install_config/aggregate_logging.adoc (hunk 7 above)
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: s/OpenShift/{product-title}
s/config/configuration

File: install_config/aggregate_logging.adoc [hunk 11]
```diff
@@ -1370,28 +1439,42 @@ Curator. Currently the `logging-curator` ConfigMap is used to configure both
...
+  edit /my/config/curator5.yaml
+  edit /my/config/actions.yaml
+$ oc delete configmap logging-curator ; sleep 1
+$ oc create configmap logging-curator \
+    --from-file=curator5.yaml=/my/config/curator5.yaml \
+    --from-file=actions.yaml=/my/config/actions.yaml \
+    ; sleep 1
 ----
 
+The next scheduled job will use this configuration.
```
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: s/will use/uses

File: install_config/aggregate_logging.adoc (hunk 9 above)
Comment (kalexand-rh) [inline install_config/aggregate_logging.adoc]: If this is a possible replacement for the previous step, then I'd turn both of them into substeps of a common ancestor step.

**Review by kalexand-rh — COMMENTED** [review]
Couple of style picks and suggestions.

**Pull request merged**